data/soapvalidator/MailClient/Calendar/Bugs/bug62674.xml

```diff
@@ -45,7 +45,17 @@
         </t:response>
     </t:test>
     
-    
+    <t:test>
```

Comment:
Please update commit comment to explain this addition.

Reply:
The request was added to get the zimbraMailHost so that it can be used later in CreateGalSyncAccountRequest.